project.clj

```diff
@@ -54,6 +54,7 @@
                  [org.clojure/tools.logging "1.1.0"]
                  [org.postgresql/postgresql "42.2.16"]
                  [org.webjars.bower/tether "1.4.4"]
+                 [org.webjars.npm/axe-core "4.0.2"]
```

Comment:
can't we have this only as a dev dependency? I don't think we need to bundle it into production jars.

Reply:
It's possible but I think there is not much harm in this one. I'm thinking that we can manually inject and test at some prod-like sites if needed for a while.

Reply:
I'd prefer it as a dev dependency for some reason, but I guess this is fine for now

Reply:
Ideally we would have the ability to run the accessibility tools with the different themes because they affect the contrast. This would mean we have to package it. And ideally the package would stay the same everywhere after dev?